### DISCUSSION TITLE
[V3] More detailed information about asserting that events are disptached.

### DISCUSSION THREAD

**Discussion by julesgraus** [livewire/livewire#5908] [7. Bugs (archived)]
Hey Guys,

Have a look at the [docs for v3 that tell you about using ```assertDispatched ```](https://livewire.laravel.com/docs/testing#events).  It doesn't mention that you can assert its parameters too. So after sourcediving, I now know how that works in V3. It's a bit different in V3 then in V2. For Example, in V2, you can pass in an array as the second parameter. In V3 you can use named parameters OR a closure as the second param. I've updated the documentation and i have created this pull request for it:

https://github.com/livewire/livewire/pull/5909

**Comment by joshhanley** [comment]
@julesgraus thanks!

**Comment by julesgraus** [comment]
I'll close this one because the PR is merged a while ago.
